package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "newsapi",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Great work on this task. You could also add a script to run your application. Forexample `"start": node index.js`